src/utils/functions.js

```diff
@@ -1,6 +1,22 @@
+import { questions, starships, vehicles } from '../constants';
+
 export const renderComponentIntoSelectedTag = (component, targetID) => {
   if (!component || !targetID) {
     return;
   }
   document.getElementById(`${targetID}`).appendChild(component);
 };
+
+export const onMenuItemClick = (category, questionPanel) => {
+  let currentQuestion = questions.peopleQuestion;
+
+  category = category.toLowerCase();
+  if (category === vehicles) {
+    currentQuestion = questions.vehiclesQuestion;
+  } else if (category === starships) {
+    currentQuestion = questions.starshipsQuestion;
+  } else {
+    currentQuestion = questions.peopleQuestion;
+  }
+  questionPanel.changeLabel(currentQuestion);
+};
```

Comment:
```suggestion
export const onMenuItemClick = (category, questionPanel) => {
  const currentQuestion = questions.peopleQuestion;

  switch(category) {
    case vehicles:
      return currentQuestion = questions.vehiclesQuestion;
      break;
    case starships:
      return currentQuestion = questions.starshipsQuestion;
      break;
    default:
      return currentQuestion = questions.peopleQuestion;
  }
  
  questionPanel.changeLabel(currentQuestion);
};
```
Idealny przykład użycia switch case :) zadbajmy,żeby przekazane category już było lowecase